store/test/database.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 const (
-	mongoTimeout = time.Second * 5
+	mongoTimeout = time.Second * 15
```

Comment:
Were you seeing timeouts? Do we have any metrics on that (i.e. how often and what calls were timing out?)

Reply:
The teardown was timing out. I've seen it happen few times locally, but never in CI.